#!/usr/bin/python3i

def raise_exception_msg(message=""):
    raise NameError(message)
